pxr/base/arch/symbols.cpp

```diff
@@ -128,7 +128,7 @@ ArchGetAddressInfo(
         }
 
         if (symbolName) {
-            *symbolName = symbol->Name ? symbol->Name : "";
+            *symbolName = symbol->Name[0] ? symbol->Name : "";
```

Comment:
Double checking ~ this fix matches documentation. https://learn.microsoft.com/en-us/windows/win32/api/dbghelp/nf-dbghelp-symfromaddr

Reply:
The fix is because symbol->Name is a "single" array, and so would always have a valid address which was the original code. The modified code I think was the intent of checking for an empty string.